aspose/src/test/java/com/testautomationguru/utility/ImageUtil.java

```diff
@@ -43,4 +42,34 @@ static void saveImage(BufferedImage image, String file) {
 			e.printStackTrace();
 		}
 	}
+
+	private static double getDifferencePercent(BufferedImage img1, BufferedImage img2) {
```

Comment:
To know if the test works as intended it would be interesting to see what the actual deviated-pct is currently on both Linux and Windows, and what the value would be for when there is a clear mismatch.